cves/CVE-2013-7014.yml

```diff
@@ -294,37 +302,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
```

Comment:
There might be a case for defensive programming as an added section. Checking for integer overflow (if that is what's happening here).